idd/Energy+.idd.in

```diff
@@ -8900,26 +8900,16 @@ Construction:InternalSource,
 
 Construction:AirBoundary,
        \memo Indicates an open boundary between two zones. It may be used for base surfaces and fenestration surfaces.
+       \memo The two adjacent zones are grouped together for solar, daylighting and radiant exchange.
        \memo When this construction type is used, the Outside Boundary Condition of the surface
        \memo (or the base surface of a fenestration surface) must be either Surface or Zone.
        \memo A base surface with Construction:AirBoundary cannot hold any fenestration surfaces.
+       \min-fields 4
   A1,  \field Name
        \required-field
        \type alpha
        \reference ConstructionNames
-  A2,  \field Solar and Daylighting Method
-       \note This field controls how the surface is modeled for solar distribution and daylighting calculations.
-       \type choice
-       \key GroupedZones
-       \key InteriorWindow
-       \default GroupedZones
-  A3,  \field Radiant Exchange Method
-       \note This field controls how the surface is modeled for radiant exchange calculations.
-       \type choice
-       \key GroupedZones
-       \key IRTSurface
-       \default GroupedZones
-  A4,  \field Air Exchange Method
+  A2,  \field Air Exchange Method
```

Comment:
So what configurations would cause this change to have results differences from version to version?  We should definitely make that clear in the transition of this.

Reply:
Any incoming 9.4 idf with `InteriorWindows` and/or `IRTSurfaces` would have diffs in 9.5. That can throw a transition warning.

Reply:
:+1: